.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+before_install:
+    - pip install tox
+    - wget https://s3.amazonaws.com/influxdb/influxdb_0.9.3_amd64.deb
+    - mkdir influxdb_install
+    - dpkg -x influxdb_0.9.3_amd64.deb influxdb_install
+    - influxdb_install/opt/influxdb/versions/0.9.3/influxd
```

Comment:
shouldn't you be able to do 
```
wget http://influxdb.s3.amazonaws.com/influxdb_0.9.3_amd64.deb
sudo dpkg -i influxdb_0.9.3_amd64.deb
sudo /etc/init.d/influxd start
```
as shown in the [docs](https://influxdb.com/docs/v0.9/introduction/installation.html)? or did that not work?